### ISSUE TITLE
Planet name is partially obscured by another planet

### ISSUE THREAD

**Issue by ImprobabilityCast** [endless-sky/endless-sky#6176]
**Describe the bug**
Other planet in Regor system partially hides the name of Windblain. Could happen in other systems, idk.

**To Reproduce**
Go to the Regor system on 20 Feb 3020

**Expected behavior**
Planet names shouldn't be blocked by other planets.

**Screenshots**
![Endless Sky 2021-08-06 09_18_55 (2)](https://user-images.githubusercontent.com/30889535/128797801-00096f6f-802f-4568-b19f-d286e2ed35a2.png)

**System:**
 - OS: Windows 10 Home, version 21H1 build 19043.1110
 - Game Source: Github release
 - Version: 0.9.14

**Link to save file**
[test test.txt](https://github.com/endless-sky/endless-sky/files/6958423/test.test.txt)

**Additional context**
N/A

**Comment by Amazinite** [comment]
Duplicate of #2340.
